packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -230,20 +250,20 @@ export const ForumEventPoll = ({event, postId, classes}: {
   // Whether or not the poll results (i.e. other users' votes) are visible.
   // They are hidden until the user clicks on "view results".
   const [resultsVisible, setResultsVisible] = useState(false)
-  const [voteCount, setVoteCount] = useState(event.voteCount)
+  const [voteCount, setVoteCount] = useState(event?.voteCount ?? 0)
```

Comment:
Since I added a refetch we could pull `voteCount` straight from `event`, but I'm keeping it in the state so that the UI is snappier